wrapanapi/scvmm.py

```diff
@@ -361,8 +361,8 @@ def update_scvmm_library(self):
         # This forces SCVMM to update Library after a template change instead of waiting on timeout
         self.logger.info("Updating SCVMM Library")
         script = """
-            $lib = Get-SCLibraryShare | where {$_.name -eq 'hyperv'}
-            SCLibraryShare -LibraryShare $lib[0] -Path VmmLibraryFiles -RunAsynchronously
+            $lib = Get-SCLibraryShare | where {$_.name -eq \'VMMLibrary\'}
+            Read-SCLibraryShare -LibraryShare $lib[0] -Path VHDs -RunAsynchronously
```

Comment:
**QUESTION**
Should `library` and `path` be a parameter of `update_scvmm_library` or is it the same for all providers?

Reply:
already have card for that. before it was always hardcoded. and scvmms kept to this naming. i'll be separate PR